packages/ui/public/deployment.json

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
is this meant to be included or ignored?

Reply:
I had it there to help the development experience, but are you suggesting adding something like a "predev" script that runs `npm run regenerate-development-example` before running `npm run dev`?

Reply:
I removed the `./public/deployment.json` from git and added the generation of it from the example to a prescript for `npm run dev`.